great-idea/css/index.css

```diff
@@ -63,4 +65,269 @@ h1, h2, h3, h4, h5 {
+	display: flex;
+}
+
+.bottom-content {
+	display: flex;
+}
+
+.bottom-content .text-content {
+	margin-right: 35px;
+}
+
+div.bottom-content {
+	margin-top: 235px;
+}
+
+
+/* Service Info Selectors */
+
+section.service-info {
+	display: flex;
+    flex-wrap: wrap;
+    align-content: space-evenly;
+    justify-content: space-between;
+    height: 860px;;
+}
+
+section.service-info .info-box {
+	background: #f4f3f4;
+    width: 48%;
+    height: 250px;
+	border: 1px solid black;
+	display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 15px;
+}
+
+
+/* Facts Selectors */
+
+.facts ul {
+	list-style-type:square;
+	list-style-position: inside;
+}
+
+.factsText ul li {
+	margin-bottom: 10px;
+}
+
+.facts {
+	display: flex;
+	justify-content: space-between;
+}
+
+.facts .factsText {
+	width: 420px;
+}
+
+.factsText p {
+	line-height: 22px;
+    margin-bottom: 20px;
+}
+
+
+/* Footer Selectors */
+
+footer p.copyright {
+	margin-top: 40px;
+	text-align: center;
+}
+
+footer section address {
+	margin-bottom: 20px;
+}
+
+footer section p {
+	line-height: 30px;
+}
+
+/* Other Selectors */
+
+
+.line {
+	margin-top: 30px;
+    margin-bottom: 40px;
+    border-top: 1px solid black;
+}
+
+.info-box button {
+	border-radius: 5px;
+    width: 122px;
+    height: 30px;
+    background-color: white;
+    border: 1px solid black;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+
+.info-box p {
+	margin-bottom: 25px;
+    line-height: 20px;
+}
+
+
+
+
+/* Media Queries  */
+
+/* If the screen size is 600px or less, */
+@media only screen and (max-width: 600px) {
+	section.service-info {
+	  height: auto;
+	  flex-flow: column nowrap;
+	  align-items: center;
+	}
+  }
+  
+/* If the screen size is 600px or less, */
+@media only screen and (max-width: 600px) {
+	section.service-info .info-box {
+		margin-bottom: 20px;
+		width: 600px;
+	}
+  }
+
+
+/* If the screen size is 600px or less, */
+@media only screen and (max-width: 600px) {
+	.info-box button {
+		margin-left: 150px;
+		margin-top: 30px;
+		font-size: 35px;
+		width: 225px;
+		height: fit-content;
+	}
+  }
+  
+/* If the screen size is 600px or less, */
+@media only screen and (max-width: 600px) {
+	.facts {
+	  height: auto;
+	  flex-flow: column nowrap;
+	  align-items: center;
+	}
+}
+	
+	
+/* If the screen size is 600px or less, */
+@media only screen and (max-width: 600px) {
+	.facts .factsText {
+		width: 600px;
+		margin-bottom: 40px;
+	}
+} 
+
+/* If the screen size is 600px or less, */
+@media only screen and (max-width: 600px){
+	footer .contact {
+		height: auto;
+		flex-flow: column nowrap;
+		align-items: center;
+		display: flex;
+    	justify-content: space-between;
+	}
+}
+
+/* If the screen size is 600px or less, */
+@media only screen and (max-width: 600px){
+	.services {
+		display: flex;
+		height: auto;
+		flex-flow: column nowrap;
+		align-items: center;
+		width: 600px;
+		margin-left: 135px;
+	}
+}
+
```

Comment:
great job adding in these media queries. we will talk about this stuff at length next week in the responsive design modules.

Reply:
I think I added to many of them or something because I talked to another fellow in the program and he told me he only used one @media query ...